crates/c/src/lib.rs

```diff
@@ -1725,7 +1733,7 @@ impl InterfaceGenerator<'_> {
         if abi::guest_export_needs_post_return(self.resolve, func) {
             uwriteln!(
                 self.src.c_fns,
-                "__attribute__((__weak__, __export_name__(\"cabi_post_{export_name}\")))"
+                "__attribute__((__export_name__(\"cabi_post_{export_name}\")))"
```

Comment:
Oh could this keep `__weak__` though? (to ensure that if needed it can be overridden without extra bindings options)

Reply:
Oops, yeah. Fixed.